.github/actions/environment/action.yml

```diff
@@ -8,4 +8,28 @@ runs:
         echo TEST_TIMEOUT=310 | tee -a $GITHUB_ENV
         echo NO_OUTPUT_TIMEOUT=320 | tee -a $GITHUB_ENV
         echo PRESERVE_ENVVARS=REPLICATION_SYNC_TIMEOUT,TEST_TIMEOUT,NO_OUTPUT_TIMEOUT | tee -a $GITHUB_ENV
+        # This switching off swap command will not work as github step
+        # run from inside github 'container' tag. Also it will fail to
+        # run from it. So running it only outside of docker container.
+        # Also on local hosts, like that we use for 'freebsd' workflow
+        # testing, 'sudo' not acceptable outside of 'freebsd' virtual
+        # machine and to avoid of hangs let's check sudo run ability
+        # with command like 'timeout 2 sudo ls /sbin/swapoff'.
+        # NOTE: To switch off swap from inside github 'container' tag
+        # additional memory flags should be added to its 'options' tag:
+        #   options: '--memory=<some value, like 7G> --memory-swap=<the same value as for memory option>'
+        free
```

Comment:
And we can call it afterwards to compare.

Reply:
Right, I thought the same, let's set it.